### DISCUSSION TITLE
How to MariaDB?

### DISCUSSION THREAD

**Discussion by JHerwig** [prisma/prisma#9328] [Q&A] [answered]
I am confused about how to use Prisma with MariaDB. Regarding the [Database reference](https://www.prisma.io/docs/reference/database-reference/supported-databases)  is MariaDB supported.

But on adding a MariaDB as datasource, I get the error `Datasource provider not known: "mariadb".`.
```
datasource db {
  provider = "mariadb"
  url      = "mariadb://johndoe:mypassword@localhost:5432/mydb?schema=public"
}
```
On reading [Prisma schema reference](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#datasource) I figured out, that MariaDB is missing.

How can I add MariaDB to Prisma to execute queries and what is needed to use Prisma migrate? 

Any help is welcome. 
Thanks

**Comment by ryands17** [comment, marked as answer]
@JHerwig 👋 
The provider and url will both be `mysql` in this case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
